backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
+(
+    subtest_id      bigserial      not null primary key,
+    test_id         bigint         not null,
+    name            varchar(256)   not null unique,
+    -- Method
+    meth            varchar(10000) null,
+    confluence_link varchar(1000)  null,
+    constraint belongs_to_test
+        foreign key (test_id) references test (test_id)
+            on update cascade
+            on delete cascade
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id bigserial                not null primary key,
+    project_id    bigint                   not null,
+    subtest_id    bigint                   not null,
+    -- This is input from the user
+    name          varchar(256)             null,
+    meta          varchar(1000)            null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name varchar(256)             not null,
+    upload_date   timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
+            on delete cascade,
+
+    constraint has_subtest
+        foreign key (subtest_id) references subtest (subtest_id)
+            on update cascade
+            -- Prevent removing subtest if there is an experiment having such subtest
+            on delete no action
+);
+
+create table measurement
+(
+    measurement_id bigserial        not null primary key,
+    experiment_id  bigint           not null,
+    -- Either compound id or name
+    compound_id    varchar(256)     not null,
+    concentration  double precision not null,
+    signal         double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete cascade
+            on update cascade
+);
+
+-- We can remove this table if we don't need anything apart from IC50
+create table analysis
+(
+    analysis_id bigserial     not null primary key,
+    description varchar(1000) null,
+    -- Analysis-specific parameters
+    parameters  json          null
+);
+
+create table analysis_outcome
```

Comment:
I wrote a short essay about "what results" are below. But now let's talk a little bit about capturing re-analysis approach!

 1. Not every measurement of each experiment will be accounted for in every analysis that is related to this experiment.
 2. Measurements used should be tracked explicitly.
 3. Relation between analyses and experiments is many-to-one

Reply:
I think our main focus for now is to make proper support of storage/hierarcy so this talks about results seems to be premature to me. I mean I don't know how we will re-analyze, whether it will be stored in the db or it's just they removed some points, see IC50 in this case and just disregard it. so I would leave this discussion for later stages when we focus on analysis and all these features about points removal/randomisation.

Reply:
so analysis is really tentative in this db I left it just in some tentative form.

Reply:
> this talks about results seems to be premature to me. I mean I don't know how we will re-analyze

They told it during the meeting while answering to one of the questions, I quote by memory:

"When the file is uploaded it's analysed automatically and the results of analysis is shown to the operator. The operator then can choose to exclude data points and click re-analyse and/or mark the results as good or bad"

So no, we do know how will reanalysing happen.